### PR TITLE
Johnson add delete functionality

### PR DESCRIPTION
**Pull request by LindseyJJ**

### CODE REVIEW

File: app/assets/stylesheets/delete/delete.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// .delete_button {
+//   padding: 9 50px;
+//   float: right;
+//   font: bold;
+// }
```
Comment (calaway) [inline app/assets/stylesheets/delete/delete.scss]: Let's delete this file if it's all commented out.

File: app/controllers/projects_controller.rb [hunk 2]
```diff
@@ -4,4 +4,9 @@ class ProjectsController < ApplicationController
   def index
     @projects = Project.all
   end
+
+  def destroy
+    @project = Project.find_by_slug!(params[:slug]).destroy
+    redirect_to projects_path
+  end
```
Comment (calaway) [inline app/controllers/projects_controller.rb]: Nice job remembering to always use `destroy` with active record, and never `delete`.

File: app/views/projects/index.html.erb [hunk 3]
```diff
@@ -29,6 +30,9 @@
             <a class="label label--fail" href="<%= project_suite_run_path(suite.project, suite, suite.latest_run, params: { status: 'fail' }) %>"><%= suite.latest_run.failing_tests %> failed</a>
           <% end %>
         </td>
+          <td>
+          <%= button_to "Delete Project", project, data: { confirm: "Are you damn sure you want to delete #{project.name} after someone worked so hard to create it?" }, method: :delete %>
```
Comment (calaway) [inline app/views/projects/index.html.erb]: Indent one more.
Also, I like the confirmation text.

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -13,3 +13,5 @@
 @import 'components/label';
 @import 'components/masthead';
 @import 'components/test';
+
+@import 'delete/delete';
```
Comment (calaway) [inline app/assets/stylesheets/application.scss]: Remove this if it didn't end up being used.

Comment (LindseyJJ) [reply app/assets/stylesheets/application.scss]: right... I forgot to do that

**Pull request merged**